packages/common.vm/tools/vm.common/vm.common.psm1

```diff
@@ -1761,7 +1761,7 @@ function VM-Install-With-Pip {
         [string] $toolName, # Example: magika
         [Parameter(Mandatory=$true)]
         [string] $category,
-        [Parameter(Mandatory=$false)]
+        [Parameter(Mandatory=$true)]
```

Comment:
the common nuspec also needs update. Please run the linter locally or check the CI status after sending the changes.

Reply:
This will cause a conflict with https://github.com/mandiant/VM-Packages/pull/1243 which also updates common. As the other PR has more changes (and is more difficult to rebase), I suggest you wait to do this change after that PR has been merged, to avoid the merge conflicts.

Reply:
Yes, from the [Wiki](https://github.com/mandiant/VM-Packages/wiki/Chocolatey-packages#version):
> **The version needs to be increased every time you modify an existing package.**

Otherwise the package won't be pushed to MyGet, as there is already a package with that version.